src/transport.ts

```diff
@@ -19,53 +19,93 @@ import * as sdp from './sdp.js'
 import { WebRTCStream } from './stream.js'
 
 const log = logger('libp2p:webrtc:transport')
+
+/**
+ * The time to wait, in milliseconds, for the data channel handshake to complete
+ */
 const HANDSHAKE_TIMEOUT_MS = 10000
-const WEBRTC_CODE: number = 280
-const CERTHASH_CODE: number = 466
 
+/**
+ * Created by converting the hexadecimal protocol code to an integer.
+ *
+ * {@link https://github.com/multiformats/multiaddr/blob/master/protocols.csv}
+ */
+export const WEBRTC_CODE: number = 280
+
+/**
+ * Created by converting the hexadecimal protocol code to an integer.
+ *
+ * {@link https://github.com/multiformats/multiaddr/blob/master/protocols.csv}
+ */
+export const CERTHASH_CODE: number = 466
+
+/**
+ * The peer for this transport
+ */
+// @TODO(ddimaria): seems like an unnessary abstraction, consider removing
 export interface WebRTCTransportComponents {
   peerId: PeerId
 }
 
 export class WebRTCTransport implements Transport {
+  /**
+   * The peer for this transport
+   */
   private readonly components: WebRTCTransportComponents
 
   constructor (components: WebRTCTransportComponents) {
     this.components = components
   }
 
+  /**
+   * Dial a given multiaddr
+   */
   async dial (ma: Multiaddr, options: WebRTCDialOptions): Promise<Connection> {
     const rawConn = await this._connect(ma, options)
     log(`dialing address - ${ma.toString()}`)
     return rawConn
   }
 
+  /**
+   * Create transport listeners no supported by browsers
```

Comment:
this sentence reads a big ambiguous and unclear, can you expand it please? E.g. `Creating transports listeners is not supported when used from within a browser`, or something like that